app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -1273,18 +1275,18 @@ $include-html-global-classes: $include-html-classes;
 // $include-html-table-classes: $include-html-classes;
 
 // These control the background color for the table and even rows
-// $table-bg: $white;
-// $table-even-row-bg: $snow;
+$table-bg: rgba( $black, .6 );
+$table-even-row-bg: rgba( $table-red, .5 );
 
 // These control the table cell border style
 // $table-border-style: solid;
-// $table-border-size: 1px;
+$table-border-size: 0px;
 // $table-border-color: $gainsboro;
 
 // These control the table head styles
-// $table-head-bg: $white-smoke;
+$table-head-bg: rgba( $table-red, .5 );
```

Comment:
`.5` should be written with a leading zero as `0.5`<br>Expected 0 spaces between parentheses instead of 1